examples/e-commerce/src/app.css

```diff
@@ -0,0 +1,376 @@
+  margin: 4rem 0;
+}
+
+/* Styles the SFFV highlightings */
+
+mark {
+  background: rgba(226, 164, 0, 0.4);
+}
+
+/* Clear refinements container */
+
+.ais-ClearRefinements {
+  align-items: center;
+  display: flex;
+}
+
+.ais-ClearRefinements-button::before {
+  content: url('./images/clear-filters.svg');
+  width: 10px;
+  height: 10px;
+  position: relative;
+  top: 1px;
+  margin-right: 5px;
+}
+
+/* Panel */
+
+.ais-Panel-header {
+  font-family: Hind, sans-serif;
+}
+
+/* Search box */
+
+.header-searchbox .ais-SearchBox {
+  height: 64px;
+  width: 740px;
+}
+
+.header-searchbox .ais-SearchBox-input {
+  background-color: #fff;
+  border-radius: 8px;
+  box-shadow: 0 4px 48px 0 rgba(0, 0, 0, 0.2);
+  font-family: Hind, sans-serif;
+  height: 64px;
+  padding-left: 58px;
+  /*
+    The "Hind" font family is vertically off-balance.
+    Adding some padding top makes it more vertically aligned.
+  */
+  padding-top: 4px;
+}
+
+.header-searchbox .ais-SearchBox-input::placeholder {
+  color: rgba(33, 36, 61, 0.24);
+  opacity: 1; /* Firefox */
+}
+
+.header-searchbox .ais-SearchBox-input:-ms-input-placeholder {
+  color: rgba(33, 36, 61, 0.24);
+}
+
+.header-searchbox .ais-SearchBox-input::-ms-input-placeholder {
+  color: rgba(33, 36, 61, 0.24);
+}
+
+.header-searchbox .ais-SearchBox-submit svg {
+  display: none;
+}
+
+.header-searchbox .ais-SearchBox-submit::before {
+  content: url('./images/submit-icon.svg');
+}
+
+.header-searchbox .ais-SearchBox-submit {
+  color: #e2a400;
+  height: 16px;
+}
+
+.ais-RefinementList-searchBox .ais-SearchBox-input {
+  font-family: Hind, sans-serif;
+  /*
+    The "Hind" font family is vertically off-balance.
+    Adding some padding top makes it more vertically aligned.
+  */
+  padding-top: 1px;
+}
+
+/* Hits */
+
+.hit {
+  color: #21243d;
+  font-size: 14px;
+  line-height: 18px;
+}
+
+.hit h1 {
+  font-size: 14px;
+}
+
+.hit-category {
+  color: #21243d;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 1;
+  margin-bottom: 8px;
+  opacity: 0.7;
+  text-transform: uppercase;
+}
+
+.hit-description {
+  margin-top: 2px;
+}
+
+.hit-description .ais-Highlight {
+  display: -webkit-box;
+  height: 100px;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.hit-image-container {
+  align-items: center;
+  display: flex;
+  height: 174px;
+  justify-content: center;
+  margin: auto;
+  width: 174px;
+}
+
+.hit-image {
+  height: auto;
+  max-width: 100%;
+}
+
+.hit-em {
+  color: #e2a400;
+  font-size: 11px;
+  font-weight: 600;
+}
+
+.hit-rating {
+  border: 1px solid rgba(226, 164, 0, 0.5);
+  border-radius: 4px;
+  margin-left: 4px;
+  padding: 0 4px;
+}
+
+/* RangeSlider */
+
+.ais-RangeSlider .noUi-target {
+  background-color: rgba(65, 66, 71, 0.08);
+  border-radius: 3px;
+  height: 3px;
+  width: calc(100% - 10px);
+  position: relative;
+  direction: ltr;
+  margin-top: 35px;
+}
+
+.ais-RangeSlider .noUi-base {
+  width: 100%;
```

Comment:
We can add `cursor: pointer;` to indicate that we can click on the bar to refine.